core/explorer/core/model.py

```diff
@@ -41,30 +46,62 @@ class BuildInputType(Enum):
 class BuildInput(BaseModel):
     """A build input to a Nix derivation"""
 
-    build_input_type: BuildInputType = Field(description="The type of build input")
-    output_path: str = Field(description="The output path of the input derivation")
+    attribute_path: str = Field(
+        description="Attribute path from the flake package set",
+    )
+    build_input_type: BuildInputType = Field(
+        description="The type of build input",
+    )
+    # None when it can't be built (e.g. wrong platform)
+    output_path: str | None = Field(
+        description="The output path of the input derivation",
+    )
 
     class Config:
         use_enum_values = True
+        alias_generator = snake_case_to_camel_case
+        allow_population_by_field_name = True
```

Comment:
Important to set this so we can use the snake_case attribute names in the model class as kwargs in the constructor.

Reply:
I think this is worth a comment in the source code.